src/Components/Navbar/Navbar.scss

```
@@ -0,0 +1,32 @@
+.Navbar {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 25px;
+  
+  .NavbarMenu {
+    display: flex;
+    align-items: center;
+    li {
+      margin: 0 15px;
+      a {
+        color: #000;
+      }
+    }
+    
+    li.myPolo a {
+      color: #dd1785;
+    }
+  }
```

```suggestion
  .NavbarMenu {
    display: flex;
    align-items: center;
    
    li {
      margin: 0 15px;
      
      a {
        color: #000;
      }
     
      &.myPolo a {
        color: #dd1785;
    }
  }
```
- 새로운 셀렉터는 분리해서 작성해주세요
- & 연산자 사용해서 묶어줄 수 있는 부분 찾아서 묶어주세요